src/main/java/com/iexec/worker/executor/TaskManagerService.java

```diff
@@ -87,199 +94,220 @@ private void unsetTaskUsingCpu(String chainTaskId) {
         log.info("Unset task using CPU [tasksUsingCpu:{}]", tasksUsingCpu.size());
     }
 
-    boolean start(String chainTaskId) {
+    ReplicateStatusUpdate start(String chainTaskId) {
```

Comment:
I think it's kind of awkward to return a ReplicateStatusUpdate in `start(..)`, `downloadApp(..)`, .. methods, can't we return somthing like 
ReplicateActionResponse {
 boolean isSuccess;
 ReplicateStatusCause statusCause;
}
Tell me if I'm wrong ;)

Reply:
Well, all "actions" finish always by updating the status of the replicate and setting the cause and details when needed, that's why I think it's coherent.

Reply:
Yes, I agree, the return structure should just be what is needed: success or not, the cause and details if needed. The result of a start or a download is not an update.